orderbook/src/orderbook.rs

```diff
@@ -69,6 +77,20 @@ impl OrderBook {
         let mut orders = self.orders.write().await;
         orders.retain(|order| has_future_valid_to(now_in_epoch_seconds, &order.order_creation));
     }
+
+    fn user_order_to_full_order(&self, user_order: OrderCreation) -> Result<Order, AddOrderError> {
```

Comment:
nit: you might wanna change the name reflecting that we don't have "user_orders" anymore